src/tools/rbd/action/Trash.cc

```diff
@@ -191,15 +196,19 @@ int do_list(librbd::RBD &rbd, librados::IoCtx& io_ctx, bool long_flag,
     return r;
   }
 
+  if (!all_flag) {
+    trash_entries.erase(remove_if(trash_entries.begin(),
+                                  trash_entries.end(),
+                                  boost::bind(is_not_trash_user, _1)),
+                        trash_entries.end());
+  }
+
+
```

Comment:
@dillaman  I'm sorry for adding a redundant blank line at here. Repushed. Thank you.